### PR TITLE
removed non-used loops vars

### PR DESCRIPTION
**Pull request by DStorck**
this removes unused variables mentioned in https://github.com/samsung-cnct/k2/issues/375

### CODE REVIEW

File: ansible/roles/kraken.nodePool/kraken.master.docker/tasks/service-account-key.yaml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ---
 - set_fact:
```
Comment (joejulian) [inline ansible/roles/kraken.nodePool/kraken.master.docker/tasks/service-account-key.yaml]: Should this be `a_cluster`, too?

Comment (coffeepac) [inline ansible/roles/kraken.nodePool/kraken.master.docker/tasks/service-account-key.yaml]: shouldn't be.  the rest of the file refers to 'cluster'

'a_cluster' is an oddity I haven't tracked down yet but there is something somewhere that is already using the variable cluster so if you try and use it when looping over cluster ansible throws a duplicate variable message.

**Comment by coffeepac** [comment]
LGTM

**Comment by coffeepac** [comment]
retest this please

**Comment by coffeepac** [comment]
retest this please

**Comment by coffeepac** [comment]
passed pipelet.  good enough for now as I don't expect the new jenkins system to pass this.

**Pull request merged**